projects/plugins/social/src/js/components/pricing-page/index.js

```diff
@@ -32,6 +32,11 @@ const PricingPage = () => {
 		updateOptions( newOption );
 	}, [ updateOptions ] );
 
+	// todo: pull pricing from wpcom
+	const firstMonthIntroOffer = 1;
+	const monthlyBasicPrice = 10;
+	const monthlyAdvancedPrice = 15;
```

Comment:
Just noting that, ideally, we should fix this ASAP but it's out of the scope of this PR.

Reply:
I agree 💯

Reply:
Is there a tracking issue or card so this doesn't get lost?

Reply:
Yes it is being tracked here: 1198218726984184-as-1204008786549223/f